src/tsserver/nodeServer.ts

```diff
@@ -381,15 +379,6 @@ export function initializeNodeSystem(): StartInput {
         sys.gc = () => global.gc?.();
     }
 
-    sys.require = (initialDir: string, moduleName: string): ModuleImportResult => {

```

Comment:
It always confused my why we had this twice, so 👍

It also feels like all but `watchDirectory` could go away above too, but, not required in this PR.